src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/nukleus.rpt

```diff
@@ -0,0 +1,460 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newTargetInputRef1 ${nuklei:newReferenceId()} # external scope
+property newTargetInputRef2 ${nuklei:newReferenceId()} # external scope
```

Comment:
Easier to have both streams use the same `newTargetInputRef` (but different `newTargetInputId`s) when integrating implementation testing with the control script that sets up the route.